src/java/org/apache/cassandra/locator/DynamicEndpointSnitch.java

```diff
@@ -378,6 +381,34 @@ public double getSeverity()
         return getSeverity(FBUtilities.getBroadcastAddressAndPort());
     }
 
+    @Override
+    public void setQuantile(double quantile)
+    {
+        if (quantile < 0.0 || quantile > 1.0 || Double.isNaN(quantile))
+        {
+            throw new IllegalArgumentException(quantile + " is not in [0..1]");
+        }
+        replicaLatencyQuantile = quantile;
+    }
+
+    @Override
+    public double getQuantile()
+    {
+        return replicaLatencyQuantile;
+    }
+
+    @Override
+    public void setQuantization(boolean enabled)
```

Comment:
If you keep millis only, please make this method either name or the parameter name reflects that it is enabling quantization to millis